Dockerfile

```diff
@@ -1,7 +1,12 @@
 FROM golang:1.4
 
+RUN apt-get update && \
+    apt-get install -y librados-dev && \
+    rm -rf /var/lib/apt/lists/*
+
 ENV DISTRIBUTION_DIR /go/src/github.com/docker/distribution
 ENV GOPATH $DISTRIBUTION_DIR/Godeps/_workspace:$GOPATH
+ENV DOCKER_BUILDTAGS include_rados
```

Comment:
>

Will this blank out existing build tags env var?

Should we be appending?

Yes, for now this is the only one build tag. I think build tags should be
appended in the Makefile if someday we need to append values to the default
ones.

>

For example, will this still work if overridden in the cli?

Which cli are we talking about? `docker build`?

Vincent